src/faction_camp.cpp

```diff
@@ -1695,13 +1695,22 @@ void basecamp::choose_new_leader()
 
 void basecamp::player_eats_meal()
 {
-    int kcal_to_eat = 3000;
+    int kcal_to_eat = 500;
     Character &you = get_player_character();
+    int stomach_left_to_engorged = 83 * to_milliliter( you.stomach.capacity(
```

Comment:
Sorry, what is... going on here? I'm not sure what value this integer is supposed to represent, or how we got an 83 in there?

I would recommend using units::volume, something like this:

```units::volume stomach_capacity_left = you.stomach.capacity(you) - you.stomach.contains();```